### ISSUE TITLE
Desktop version does not work offline

### ISSUE THREAD

**Issue by ThomasWallnerYoummday** [ethereum/remix-desktop#164]
Please give us a desktop version, which does not automatically die when an update is available.

As developers, we depend on a stable development environment.
As soon as a new version is available, you're forced to switch and if something doesn't work there, you're lost.
Version 1.3.5 is once again an absolute disaster and there is no way to work properly with Remix.

**Comment by yann300** [comment]
yes we are aware of the difficulty with the desktop version. We will be pushing some improvements in the next following weeks, thanks anyway for the head up.

**Comment by SadPencil** [comment]
An offline version is appreciated. It would solve https://github.com/ethereum/remix-desktop/issues/167
